packages/utils/src/lib/reports/formatting.ts

```diff
@@ -103,21 +103,24 @@ export function formatSourceLine(
     : `${startLine}`;
 }
 
-export function getGitHubLink(
+export function formatGitHubLink(
   file: string,
   position: SourceFileLocation['position'],
-  outputDir: string,
 ): string {
   const baseUrl = getGitHubBaseUrl();
-  const fullPath = pathPosix.join(outputDir, file);
-
   if (!position) {
-    return `${baseUrl}/${fullPath}`;
+    return `${baseUrl}/${file}`;
   }
+  const { startLine, endLine, startColumn, endColumn } = position;
 
-  const { startLine, endLine } = position;
-  const line = endLine ? `#L${startLine}-L${endLine}` : `#L${startLine}`;
-  return `${baseUrl}/${fullPath}${line}`;
+  const start = startColumn ? `L${startLine}C${startColumn}` : `L${startLine}`;
+  const end = endLine
+    ? endColumn
+      ? `L${endLine}C${endColumn}`
+      : `L${endLine}`
+    : '';
+  const lineRange = end ? (start === end ? start : `${start}-${end}`) : start;
```

Comment:
(Nitpick) This would be a little easier to follow:

```suggestion
  const lineRange = end && start !== end ? `${start}-${end}` : start;
```